gazebo/Dockerfile

```
@@ -20,7 +20,7 @@ COPY ./gazebo/movai-entrypoint.sh /usr/local/bin/movai-entrypoint.sh
 RUN wget --progress=dot:giga "https://packages.osrfoundation.org/gazebo.gpg" -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg && \
     echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null  && \
     apt-get update && \
-    apt-get install ignition-fortress=1.0.3-1~bionic -y --no-install-recommends && \
+    apt-get install libignition-gazebo6-dev=6.12.0-1~focal ignition-fortress=1.0.3-2~focal -y --no-install-recommends && \
```

Do we know why we are fixing this specific library? `libignition-gazebo6-dev`

---

gpu sensor simulation is different and matches the developments made on the the sdf and flows.
It contains also a bug fix from Carlos about the dense clouds on lidars and etc.

---

Nice. Do we need the specific version? When we run the build, wouldn't it get the latest one?

---

We tested the latest one (I think is this one), we should freeze it for avoiding future issues.

---

Ok. Unfortunately, they keep only one version of the package so in the future we won't have any other versions. Currently, they only have the version mentioned here available and no history. This is true for ignition-fortress as well. 

I'm suggesting we remove it and keep only `ignition-fortress`

```suggestion
    apt-get install ignition-fortress -y --no-install-recommends && \
```

---

@MithunKinarullathil  you are right , here are the dependencies:
`Depends: libignition-cmake2-dev (>= 2.12.1), libignition-common4-dev (>= 4.5.0), libignition-fuel-tools7-dev, libignition-gazebo6-dev (>= 6.9.0), libignition-gui6-dev (>= 6.4.0), libignition-launch5-dev (>= 5.1.0), libignition-math6-dev (>= 6.10.0), libignition-math6-eigen3-dev (>= 6.10.0), libignition-msgs8-dev (>= 8.4.0), libignition-physics5-dev (>= 5.1.0), libignition-plugin-dev (>= 1.2.1), libignition-rendering6-dev (>= 6.3.1), libignition-sensors6-dev (>= 6.3.0), libignition-tools-dev (>= 1.4.1), libignition-transport11-dev, libignition-utils1-cli-dev (>= 1.4.0), libignition-utils1-dev (>= 1.4.0), libsdformat12-dev (>= 12.4.0), python3-ignition-math6, python3-ignition-gazebo6`

---

I prefer keeping the package version to monitor any unwanted update